pandora-client-web/src/editor/parsing.ts

```diff
@@ -24,19 +24,33 @@ function ParseFloat(input: string): number {
 }
 
 export function SerializeAtomicCondition(condition: Immutable<AtomicCondition>): string {
-	if ('bone' in condition && condition.bone != null)
+	if ('bone' in condition) {
+		Assert(condition.bone != null);
 		return `${condition.bone}${condition.operator}${condition.value}`;
-
-	if ('module' in condition && condition.module != null)
+	} else if ('module' in condition) {
+		Assert(condition.module != null);
 		return `m_${condition.module}${condition.operator}${condition.value}`;
-
-	if ('armType' in condition && condition.armType != null)
+	} else if ('armType' in condition) {
+		Assert(condition.armType != null);
 		return `hand_${condition.armType}_${condition.side}${condition.operator}${condition.value}`;
-
-	AssertNever();
+	} else if ('attribute' in condition) {
+		Assert(condition.attribute != null);
+		return `attribute ${condition.attribute}`;
```

Comment:
This serialization won't work, as during deserialization, things are split by space first, as space splits condition and image... I really need to improve this :sweat_smile: 
```suggestion
		return `a_${condition.attribute}`;
```